frontend/src/tabs/authorship.pug

```diff
@@ -30,13 +30,32 @@
             option(v-bind:value='false') Ascending
           label.medium-font order
       .searchbox
-        input.radio-button--search(type="radio", value="search", v-model="filterType")
-        form.file-picker.mui-form--inline(onsubmit="return false")
-          input#search.medium-font(type="search", placeholder="Filter by glob", v-on:change="updateFilterSearch",
-            v-on:click="indicateSearchBar")
-          button#submit-button.medium-font(type="submit", v-on:click="indicateSearchBar") Filter
+        input.radio-button--search(
+          type="radio",
+          value="search",
+          v-model="filterType",
+          v-on:change="indicateSearchBar(); updateSearchBarValue();"
+        )
+        .file-picker.mui-form--inline
```

Comment:
Is the `form` removed on purpose?

Reply:
yup, that was the cause of this, I removed it to prevent writing `v-on:xx.prevent` all over since we aren't using it as a form per se

> Seems like the UI changed slightly after the merge conflict (the checkboxes). Can fix it?
> <img alt="Screen Shot 2020-04-20 at 8 11 08 PM" width="869" src="https://user-images.githubusercontent.com/32864116/79750216-2715a980-8343-11ea-86d6-94de2c6331d1.png">